Bootstrap-Policies/README.md

```diff
@@ -1,42 +1,96 @@
+
+Whether you use these example documents or not, the most important
+thing for your nascent TLP or TLWG is to establish a charter.  A
+charter is a *statement of purpose*, a *list of responsibilities*,
+and a *list of initial membership*.
+
+### Statement of Purpose
+
+What is the goal of your TLP or TLWG?
+
+
+### List of Responsibilities
+
+The list of responsibilities should be specific. Once established,
+these responsibilities should not change, and therefore should not
+be broad or subjective. 
+
+If the responsibilities described in the draft charter are currently
+undertaken by another Planetary Software TLP or TLWG, then a
+discussion is required between the existing entity and your new
+proto-TLP or proto-TLWG.  The result will either be that you will
+remove that responsibility from your list, or the existing entity
+will alter their charter to accomodate the change of responsibility.
+
+
+### List of Initial Membership
+
+Those like-minded individuals referenced in Item #2 above are 
+referred to as the Tehcnical Committee (TC) for the TLP or TLWG.
+
+A TC should have at least 3 initial members to start the work and
+get the application submitted to the Planetary Software TSC, but
+it could be fewer.  It will need 3 members in order to graduate
+from Incubation. These should be individuals already undertaking
+the work described in the charter.
+
+Are you concerned about being able to get 3 people?  If your
+application is accepted into Incubation, the Planetary Software TSC
+will help you find a strategy for growing your TC to the needed
+size.
 
-You can submit the WG charter for ratification by sending
-a Pull Request that adds the charter to the Working-Groups.md document,
-which adds it to the list of current Working Groups. The WG is considered
-to be chartered once that PR lands. Once ratified the list of members should be
-maintained in the Working Group's README.
 
 ### Base Policies for new Working Groups
 
-Once the TC ratifies a charter, the WG inherits the policies established by the
-TC for governance, contribution, conduct and an MIT LICENSE. WGs are free to
-change these documents through their own governance process but should retain
-at least the spirit of the original versions. The template documents provided
-in this directory are provided to help bootstrap that process:
-
-* [Charter Template][]
-* [Governance Template][]
-* [Code of Conduct Template][]
-* [Moderation Policy Template][]
-
-[Charter Template]: Charter.md
-[Governance Template]: Governance.md
-[Code of Conduct Template]: Code-Of-Conduct.md
-[Moderation Policy Template]: Moderation-Policy.md
+Once your TC ratifies a charter (whether you used the [Charter
+Template] in this directory or created your own), you could go
+straight to writing your application to the Planetary Software
+Organization.
+
+However, to get out of Incubation, you will also need to establish
+some other documents, like the ones in this directory, so you could
```

Comment:
`some other documents` is ambiguous. We should provide an explicit list here or link to an explicit list in the Project Lifecycle document.

Reply:
It is **_intentionally_** ambiguous, because I don't like maintaining lists: they get out of sync too damn quickly.  Not every TLP and TLWG will need *every* document in the templates directory.  Their presence in the Bootstrap-Policies directory is list enough, and it is up to the proposer and their mentor(s) to decide what is important.  Let's not be overly prescriptive.

Reply:
I think this is a place where having explicit requirements to get out of incubation is valuable. If we leave this open to case-by-case decisions, it undermines the meaning of being a TLP or TLWG.

The lifecycle document articulates requirements that are a good list.

Maybe the problem is focusing on documents when the lifecycle document does not. Here's a stab at some different text.
> However, to get out of Incubation, there are other [requirements](https://github.com/planetarysoftware/TSC/blob/master/Project-Lifecycle.md#top-level-project-and-working-group-requirements). The documents in this directory are designed to help your TLP or TLWG meet those requirements and are another good starting point.

Reply:
Okay, I like this.  Changed.